src/test/scala/org/broadinstitute/hail/methods/InfoScoreSuite.scala

```diff
@@ -0,0 +1,54 @@
+package org.broadinstitute.hail.methods
+
+import org.broadinstitute.hail.utils._
+import org.broadinstitute.hail.driver._
+import org.testng.annotations.Test
+import org.broadinstitute.hail.SparkSuite
+
+class InfoScoreSuite extends SparkSuite {
+  @Test def test() {
+    val genFile = "src/test/resources/infoScoreTest.gen"
+    val sampleFile = "src/test/resources/infoScoreTest.sample"
+    val truthResultFile = "src/test/resources/infoScoreTest.result"
+
+    var s = State(sc, sqlContext)
+    s = ImportGEN.run(s, Array("-s", sampleFile, genFile))
+    s = InfoScore.run(s, Array.empty[String])
+
+    val truthResult = sc.parallelize(hadoopConf.readLines(truthResultFile)(_.map(_.map { line =>
```

Comment:
probably better to collect the hail result rather than parallelize the local result.

also, you could have achieved this by using `sc.readLines` rather than `sc.parallelize(hadoopConf.readLines(...`